### PR TITLE
Adding Azure Lab Services generated sdk

### PR DESCRIPTION
**Pull request by rogerbest1999**
## Description

Adding Lab Services to PowerShell SDK

## Checklist

- [x ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x ] The title of the PR is clear and informative
- [x ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - Azure/azure-powershell-cmdlet-review-pr#1052
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by BethanyZhou** [comment]
Design doc: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1052

**Comment by BethanyZhou** [comment]
Hi @RogerBestMsft , please check following issues:
```
  "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
  "Az.LabServices","Add-AzLabServicesUserQuota","Add-AzLabServicesUserQuota","1","8100","Add-AzLabServicesUserQuota Does not support ShouldProcess but the cmdlet verb Add indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
  "Az.LabServices","New-AzLabServicesSchedule","New-AzLabServicesSchedule","1","8410","Parameter Notes of cmdlet New-AzLabServicesSchedule does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
  "Az.LabServices","Redeploy-AzLabServicesVM","Redeploy-AzLabServicesVM","1","8300","Redeploy-AzLabServicesVM uses the verb 'Redeploy', which is not on the list of approved verbs for PowerShell commands. Use the cmdlet 'Get-Verb' to see the full list of approved verbs and consider renaming the cmdlet.","Consider renaming the cmdlet to use an approved verb for PowerShell."
  "Az.LabServices","Start-AzLabServicesUserVM","Start-AzLabServicesUserVM","1","8100","Start-AzLabServicesUserVM Does not support ShouldProcess but the cmdlet verb Start indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
  "Az.LabServices","Stop-AzLabServicesUserVM","Stop-AzLabServicesUserVM","1","8100","Stop-AzLabServicesUserVM Does not support ShouldProcess but the cmdlet verb Stop indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
  "Az.LabServices","Sync-AzLabServicesLabUsers","Sync-AzLabServicesLabUsers","1","8400","Sync-AzLabServicesLabUsers uses the noun 'AzLabServicesLabUsers', which does not follow the enforced naming convention of using a singular noun for a cmdlet name.","Consider using a singular noun for the cmdlet name."

```

**Comment by BethanyZhou** [comment]
For Redeploy-AzLabServicesVM, which is not on the list of approved verbs for PowerShell commands. I recommend to rename it as Invoke-AzLabServicesVMRedeployment.

File: src/LabServices/readme.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  - https://github.com/Azure/azure-rest-api-specs/blob/6d7653ffd37cdc781e16202306567e355b45ebf8/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/Labs.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/6d7653ffd37cdc781e16202306567e355b45ebf8/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/OperationResults.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/6d7653ffd37cdc781e16202306567e355b45ebf8/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/Schedules.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/6d7653ffd37cdc781e16202306567e355b45ebf8/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/Users.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/6d7653ffd37cdc781e16202306567e355b45ebf8/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/VirtualMachines.json
```
Comment (isra-fel) [inline src/LabServices/readme.md:61]: Save some repeat
```suggestion
branch: 6d7653ffd37cdc781e16202306567e355b45ebf8
input-file:
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/Images.json
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/LabPlans.json
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/LabServices.json
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/Labs.json
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/OperationResults.json
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/Schedules.json
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/Users.json
  - $(repo)/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-10-01-preview/VirtualMachines.json
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Review by isra-fel — APPROVED** [review]
Looks good!

**Comment by isra-fel** [comment]
BTW, could you give me a brief changelog text that describes the initial release? It'll appear on powershellgallery.com

Something like
```
This is the initial version of Az.LabServices. It contains the following features:
- xxx
- xxx
```

**Pull request merged**